src/feature/ShapeContext.cpp

```diff
@@ -29,7 +29,8 @@ Descriptor* ShapeContext::clone() const{
 }
 	
 void ShapeContext::getFlatDescription(std::vector<double>& description) const {
-	description.clear();
+    // TODO: This is a temporary solution to make FLANN use easy:
```

Comment:
Because it makes it convenient to dump all the descriptors of a scan in one raw array that can be consumed by FLANN.